google-cloud-datastore/Gemfile

```diff
@@ -10,4 +10,4 @@ gem "rake"
 
 # Pin minitest to 5.11.x to avoid warnings emitted by 5.12.
 # See https://github.com/googleapis/google-cloud-ruby/issues/4110
-gem "minitest", "~> 5.11.3"
+gem "minitest"
```

Comment:
Please still provide a version restriction for the minitest gem. `~> 5.14` would probably work. Also, remove the above comment.

Reply:
I think you could remove this from Gemfile completely, and use the dependency as specified in gemspec instead. This gem was only listed here to work around the expectations warnings added in 5.12, which this PR already addresses.

Reply:
Good catch @blowmage, thanks!